x-pack/libbeat/management/managerV2.go

```diff
@@ -385,8 +398,12 @@ func (cm *BeatV2Manager) unitListen() {
 			cm.UpdateStatus(lbmanagement.Stopping, "Stopping")
 			return
 		case change := <-cm.client.UnitChanges():
+			cm.logger.Debugf(
```

Comment:
I am open to doing this at the Info level as a type of audit log for when changes happened. If we only log actual changes, it should occur that frequently.

I'm not sure we need the `[features]` prefix either since this doesn't seem specific to the feature flags.